include/rnp/rnp.h

```diff
@@ -1239,6 +1251,17 @@ RNP_API rnp_result_t rnp_uid_is_revoked(rnp_uid_handle_t uid, bool *result);
 RNP_API rnp_result_t rnp_uid_get_revocation_signature(rnp_uid_handle_t        uid,
                                                       rnp_signature_handle_t *sig);
 
+/**
+ * @brief Remove userid with all of it's signatures from the key
```

Comment:
Shouldn't it be "its" in English?

Reply:
@rrrooommmaaa Yeah, thanks. Sometimes quick-typing produces weird results. Will fix it.

Reply:
Actually, there are also occurrences in other places. Should we fix them or leave as they are?

Reply:
@rrrooommmaaa It would be better to issue a separate PR, as this one is not related to language fixes.